helpdesk_mgmt/models/helpdesk_ticket.py

```diff
@@ -78,6 +78,8 @@ def _read_group_stage_ids(self, stages, domain, order):
         ('normal', 'Default'),
         ('done', 'Ready for next stage'),
         ('blocked', 'Blocked')], string='Kanban State')
+    sequence = fields.Integer(string='Sequence', index=True, default=10,
```

Comment:
with the default value you basically turn off sorting by number. Better have no default, then users who use the sequence can have it, but it will be null for others and it reverts to number

Reply:
If sequence is 10 for every ticket (when users don't use the sequence), tickets will be sorted by `number`, right?
(I followed the same approach as `project.task`)

Reply:
good point, ignore then